gateway-core/components/org.wso2.carbon.gateway.core/src/main/java/org/wso2/carbon/gateway/core/flow/mediators/builtin/flowcontrollers/filter/Scope.java

```diff
@@ -25,6 +25,7 @@
     // TODO: I think we need to capitalize all of them.
     HEADER,
     MessageBody,
-    MessageHeader
+    MessageHeader,
+    BODY
```

Comment:
Any reason why some of the constants are in camel case and others in upper case?